#!/usr/bin/env python
# Predator configuration

PACKAGE = 'pandora_vision_predator'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# Example : gen.add("int_param", int_t, 0, "An In.....", 50, 0, 100)
# name type level description def min max


#------------------------Threshold parameters------------------------
gen.add("show_predator_image", bool_t, 0,"", False)

exit(gen.generate(PACKAGE, "predator_node", "predator_cfg"))
